packages/google_maps_flutter/google_maps_flutter/example/lib/main.dart

```diff
@@ -28,14 +31,19 @@ import 'scrolling_map.dart';
 import 'snapshot.dart';
 import 'tile_overlay.dart';
 
+/// Map ID is required for some examples to use advanced markers.
+const String? _mapId = null;
```

Comment:
I'm having trouble reconciling the comment and the line of code; if it's required for the examples, why is it null?

Is the idea that someone should put in their own map ID here and run the modified example? If so, the comment should be explicit about that.

Reply:
Yes, the idea is that someone should put their own ID. I've updated the comment to

```dart
/// Place your map ID here. Map ID is required for pages that use advanced
/// markers.
```

Hope it makes it clear what needs to be done and why.